standingscontroller.cpp

```diff
@@ -74,26 +76,26 @@ void StandingsController::initStandings() {
                     continue;
```

Comment:
Shouldn't we update `timeLastTry` in the if above (the one starting at line 59) as well?

Reply:
Nope because if the submission is already processed that mean that it was rejudged but it still submitted on the same time of last judgement, I already get that scenario while testing yesterday (judghost need libcgoup) and no need for that.

Reply:
Wow you know my code better than I do, lol

Reply:
I have read it twice that's why xD